deploy/osd-project-request-template/00-template.project-request.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: template.openshift.io/v1
```

Comment:
Hmm.. Having this object in a SSS with sync mode will clobber any changes the customer makes to it.

Might need to think how we can deploy objects with a different `resourceApplyMode` in the context of MCC

Reply:
That's a good point. Do you think it would help to put this template in with an operator's deployment manifests instead? Like in rbac-permissions-operator?

In v3, we had customers set an [annotation](https://docs.openshift.com/dedicated/3/admin_guide/managing_projects.html#managing-limits-and-quota-on-project-resources) on the template in order to prevent overwrites. That way they could choose to keep up-to-date with our changes, or go their own way. CC @jewzaam  any thoughts?